src/amo/components/UserProfile/styles.scss

```diff
@@ -120,3 +120,13 @@ $avatar-size: 64px;
 .UserProfile-reviews {
   margin-top: 0;
 }
+
+.UserProfile-review--is-reply {
+  background-color: transparentize($blue-50, 0.95);
```

Comment:
The more features we add, the more we duplicate styles. I would like to move towards unifying these components soon.